docs/embeddedclustered.adoc

```diff
@@ -20,7 +20,7 @@ We are going to add the following lines of code to our onStart method
         cacheManager = new DefaultCacheManager(global.build());
 ----
 
-Copy the and replace the onStart method in our ScoreService.java
+Replace the onStart method in our ScoreService.java
```

Comment:
We need to specify copy as well, since lots of times users won't know when to copy from the different code examples given in the labs.

Reply:
but "copy the and replace" sounds weird.
You meant copy and rename the onStart or copy and comment-out
and add the new onStart?

Reply:
The onStart method already exists? 
Copy is to copy from the web browser, and replace is to replace it in the CRW. 
I agree in many cases we will have people who understand it easily, however in some cases they might/will not. When the workshop runs there could be multiple people doing it.. often 50+, from all sorts for backgrounds, we need to be specific

Reply:
the onStart is already added from the last chapter (without the cluster but local) so it is a change to that method

Reply:
added a more detailed instruction